Dockerfile.centos

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2012-2016 Red Hat, Inc
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#   Dharmit Shah  - Initial implementation
+#   Mario Loriedo - Improvements
+#
+# To build it, run in the repository root:
+#  `docker build -t registry.centos.org/eclipse/che-server -f Dockerfile.centos .`
```

Comment:
@benoitf I think that's to be consistent with the comments on other Dockerfiles https://github.com/eclipse/che/blob/master/Dockerfile#L21. The registry and repository correspond to the ones used by the CI. And that's still a command that a user can execute safely on its machine. If a user want to push the resulting image (to the docker hub or to another registry) he will need to re-tag it anyway with his <registry>/<repo>/che-server.